quinn/src/endpoint.rs

```diff
@@ -46,6 +46,14 @@ pub struct Endpoint {
     pub(crate) default_client_config: Option<ClientConfig>,
     runtime: Arc<dyn Runtime>,
 }
+#[derive(Debug, Default, Copy, Clone)]
```

Comment:
This should be defined further down, probably between `impl Endpoint` and `struct EndpointDriver(..)`.